app/build-js.sh

```diff
@@ -24,21 +24,21 @@ fi
 # Add navitron & its dependencies to the app bundle
 pushd $ROOT/node_modules/navitron
 bower install
-cp bower_components/requirejs/require.js $MYPATH/scaffold-www/js
-cp bower_components/plugin/src/js/plugin.js $MYPATH/scaffold-www/js
-cp bower_components/mobify-velocity/velocity.min.js $MYPATH/scaffold-www/js
-cp dist/navitron.min.js $MYPATH/scaffold-www/js
+cp bower_components/requirejs/require.js $MYPATH/app-www/js
+cp bower_components/plugin/src/js/plugin.js $MYPATH/app-www/js
+cp bower_components/mobify-velocity/velocity.min.js $MYPATH/app-www/js
+cp dist/navitron.min.js $MYPATH/app-www/js
 popd
 
 pushd $ROOT/node_modules/jquery/dist
```

Comment:
I don't think we need to be doing a `pushd` here.  We can just specify this path in the `cp` command below... saves us doing a `pushd` and `popd` right after.